runtime/opt/taupage/bin/push-audit-logs.py

```diff
@@ -11,11 +11,13 @@
 import sys
 import time
 
-from taupage import configure_logging, get_config, get_token, get_boot_time
+from base64 import b64encode
+
+from taupage import configure_logging, get_config, get_boot_time
 
 
 def push_audit_log(config: dict, instance_logs_url, account_id, region, instance_id, boot_time, fn, compress=False):
-    token = get_token(config, 'taupage', ['uid']) or {}
+    userAndPass = b64encode(bytes(config.get('logsink_username').append(':').append(config.get('logsink_password')), encoding='ascii')).decode("ascii") or ''
```

Comment:
because a lot of app, do not need any credentials. But we still need their logs.